rpp30-boc-meltdown/biteshare
# Withdarw access to current session for denied/removed guests

## Description

- When denied/removed, the guest would be taken back to the explore menu and could not see the content on current session tab
- Cleaning ups

## How to test

Test with both a host view and a guest view
- verify that when the guest is denied, current session tab doesn't display info about the active session
- verify that when the guest is allowed then removed, current session tab doesn't display info about the active session

## Notes


## Issue tracking

https://trello.com/c/e9Z3W2yE
